lib/identity_pki/token.ex

```diff
@@ -7,7 +7,11 @@ defmodule IdentityPki.Token do
       if expires_at do
         Map.put(data, :expires_at, expires_at)
       else
-        data
+        expires_at =
+          NaiveDateTime.utc_now()
+          |> NaiveDateTime.add(300, :second)
+
+        Map.put(data, :expires_at, expires_at)
```

Comment:
**Suggestion:** Since this happens in both branches of the conditional (same as line 8) is it worth pulling out?

Reply:
Yep, good call. The `expires_at` could be handled similarly to the `secret_key` and be the default value used when fetching from options.

This also makes it possible to skip setting `expires_at` by explicitly calling `Token.box(%{}, expires_at: nil)`, should we need to support that behavior. Not passing a value will still default to 5 minutes though. Changed in 5bb36ff76798b88b67653fef825d27a34e266953